### DISCUSSION TITLE
Is it possible to test on a source not related to a specific source column?

### DISCUSSION THREAD

**Discussion by jenniferkwanaft** [dbt-labs/dbt-core#9424] [Q&A]
I want to run a generic test that I've written against a source. The test doesn't depend on any specific column; the SQL selects against the entire table. `dbt test` isn't picking up the test. Is this possible?

**Comment by graciegoheen** [comment]
Yes this is possible. Here's an example from our dbt-utils package -> https://github.com/dbt-labs/dbt-utils?tab=readme-ov-file#equal_rowcount-source
